test/WebJobs.Script.Tests/EndToEndTestFixture.cs

```diff
@@ -36,6 +37,8 @@ protected EndToEndTestFixture(string rootPath, string testId)
                 FileLoggingEnabled = true
             };
 
+            ApiHubTestHelper.SetDefaultConnectionFactory();
```

Comment:
This will impact all other tests as well. Can it be moved somewhere else which is specific to ApiHUb?